docs/Administration.rst

```diff
@@ -4639,13 +4642,128 @@ You can review the list of kudos received and sent from the tab "Kudos List" of
 
 |image336|
 
-
 You can filter per period type and select the period in which you want to see all sent (and recieved) kudos.
 
 |image335|
 
 
 
+.. _ManagingRewards:
+
+================
+Managing Rewards
+================
+
+You can manage the rewards sending from "Reward administration" page. 
+To access to this interface, click on "Administration" menu from the top navigation, then "Reward" > "Reward" from the drop-down menu.
+The first thing to do is to setup how rewards will be calculated from the configuration tab. 
+Then, from "Pools" tab you can create pools and assign users to them according to their domains. 
+Since the use of the platform may not be the same from one team to another, and in order to ensure a fair distribution of rewards, you can divide users by pools according to their teams. 
+This will put the members of the same team in competition together but not with all the users of the platform.
```

Comment:
```suggestion
Rewards will be distributed comparatively to members within the same pool. Hence, members of the same pool will compete together for rewards rather than with all the users of the platform.
```